Userland/Libraries/LibJS/Runtime/VM.cpp

```diff
@@ -823,6 +823,29 @@ ThrowCompletionOr<void> VM::link_and_eval_module(Module& module)
     return {};
 }
 
+static String resolve_filepath(StringView filepath, StringView module_type)
+{
+    auto extensions = Vector<StringView, 2> { "js"sv, "mjs"sv };
+    if (module_type == "json"sv)
+        extensions = { "json"sv };
```

Comment:
Maybe it is just me or that JSON modules are still new, but with json this behavior feels slightly more unexpected especially the `folder/index.json` case.

Not really a problem just something that I felt here.

Reply:
I definitely want it for consistency, but I agree - it's less expected than for plain JS files.
FWIW, the codebase I work on at $work is also configured to do this (some webpack magic), so I'm quite used to seeing `import ... from "./Foo.fixtures";` which actually is `Foo.fixtures.json`.

Reply:
Yeah it seems fine, just JSON modules being new to me